src/main/java/com/laba/solvd/pharmacy/mybatis/implDAO/AddressDAO.java

```diff
@@ -0,0 +1,79 @@
+package com.laba.solvd.pharmacy.mybatis.implDAO;
+
+import com.laba.solvd.pharmacy.interfaces.IAddressDAO;
+import com.laba.solvd.pharmacy.model.Address;
+import com.laba.solvd.pharmacy.utils.MyBatisConfig;
+import org.apache.ibatis.session.SqlSession;
+import org.apache.ibatis.session.SqlSessionFactory;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.util.List;
+
+public class AddressDAO implements IAddressDAO<Address> {
+    private static final Logger LOGGER = LogManager.getLogger(AddressDAO.class.getName());
+    private static SqlSessionFactory sqlSessionFactory;
```

Comment:
please use same formatting in all classes
```suggestion
public class AddressDAO implements IAddressDAO<Address> {

    private static final Logger LOGGER = LogManager.getLogger(AddressDAO.class.getName());
    private static SqlSessionFactory sqlSessionFactory;
```